lib/mixlib/install/product.rb

```diff
@@ -193,8 +193,10 @@ def lookup(key, version = :latest)
 
   product "automate" do
     product_name "Chef Automate"
-    package_name "delivery"
-    ctl_command "delivery-ctl"
+    package_name "automate"
+    ctl_command do |v|
+      v < version_for("4.4.4") ? "delivery-ctl" : "automate-ctl"
```

Comment:
yeah it is a place holder for the version when automate-ctl was included in the delivery package